test/cookbooks/osl-matrix-test/recipes/element.rb

```diff
@@ -0,0 +1 @@
+osl_element 'talk.osuosl.intnet'
```

Comment:
Please use `example.org` as the domain for testing all of these.
```suggestion
osl_element 'talk.example.org'
```

Reply:
Please fix this.